### PR TITLE
Fixed conjob and added database instructions

### PR DESCRIPTION
**Pull request by november781**
Cron needed its logging directory made in the image (decided to add that to the Dockerfile and let it be in the image instead of doing it in the entrypoint script)

The default_cronjob needed to have sudo so it could make a local php call instead of root

Since it looks like the static database still needs to be imported, I added the .zip and instructions for it. Solves #12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,8 +7,22 @@ Dockerfile for running [Pathfinder](https://github.com/exodus4d/pathfinder), the
 
 # Setup
 1. Navigate to your Pathfinder page, go through setup.
-2. Create the databases, import from ESI at the bottom.
-3. Restart your container with `SETUP=False`
-4. You're live!
+2. Create the databases using the database controls in the setup page.
+3. [Import static databse.](#Importing-static-databse)
```
Comment (porowns) [inline README.md]: typo here

File: README.md [hunk 2]
```diff
@@ -7,8 +7,22 @@ Dockerfile for running [Pathfinder](https://github.com/exodus4d/pathfinder), the
...
+4. Import from ESI at the Cronjob section of the setup page.
+5. Build Systems data index under `Build search index` in the Administration section of the setup page.
+5. Restart your container with `SETUP=False`.
+6. You're live!
+
+# Importing staic database
+1. Download the static database.
```
Comment (porowns) [inline README.md]: Should just specify command: `wget https://github.com/exodus4d/pathfinder/raw/master/export/sql/eve_universe.sql.zip`

File: README.md [hunk 3]
```diff
@@ -7,8 +7,22 @@ Dockerfile for running [Pathfinder](https://github.com/exodus4d/pathfinder), the
...
+2. Unzip `eve_universe.sql.zip`.
```
Comment (porowns) [inline README.md]: Should add command: `unzip eve_universe.sql.zip`

File: README.md [hunk 4]
```diff
@@ -7,8 +7,22 @@ Dockerfile for running [Pathfinder](https://github.com/exodus4d/pathfinder), the
...
+3. Copy the databse into the db container `docker cp eve_universe.sql {full container name: eg. pathfinder-docker_db_1}:/eve_universe.sql`.
+4. Enter the db container `docer-compose exec db /bin/bash`.
```
Comment (porowns) [inline README.md]: Typo here... `docker-compose exec db /bin/bash`

File: README.md [hunk 5]
```diff
@@ -7,8 +7,22 @@ Dockerfile for running [Pathfinder](https://github.com/exodus4d/pathfinder), the
...
+3. [Import static databse.](#Importing-static-databse)
+4. Import from ESI at the Cronjob section of the setup page.
+5. Build Systems data index under `Build search index` in the Administration section of the setup page.
+5. Restart your container with `SETUP=False`.
+6. You're live!
+
+# Importing staic database
+1. Download the static database.
+2. Unzip `eve_universe.sql.zip`.
+3. Copy the databse into the db container `docker cp eve_universe.sql {full container name: eg. pathfinder-docker_db_1}:/eve_universe.sql`.
```
Comment (porowns) [inline README.md]: Typo here... `database`

File: README.md [hunk 6]
```diff
@@ -7,8 +7,22 @@ Dockerfile for running [Pathfinder](https://github.com/exodus4d/pathfinder), the
...
+4. Enter the db container `docer-compose exec db /bin/bash`.
+5. Make sure you have completed step 2 in Setup.
```
Comment (porowns) [inline README.md]: This line isn't needed imo, they're following steps

File: README.md [hunk 7]
```diff
@@ -7,8 +7,22 @@ Dockerfile for running [Pathfinder](https://github.com/exodus4d/pathfinder), the
...
+6. Link static databse `mysql -u root -p eve_universe< eve_universe.sql`.
```
Comment (porowns) [inline README.md]: Typo

**Review by november781 — COMMENTED** [review]
Fixed typos

**Review by porowns — CHANGES_REQUESTED** [review]

**Review by november781 — COMMENTED** [review]
added those commands as requested

**Comment by november781** [comment]
all changes should be up-to-date and ready for merge

**Review by porowns — APPROVED** [review]

**Pull request merged**